.gitignore

```diff
@@ -18,6 +18,8 @@ build/Release
 
 # Users Environment Variables
 .lock-wscript
+yarn.lock
```

Comment:
As opposed to ignoring the lock file, I would suggest adding a `.gitattributes` file and then treating it as a binary. Removes the merge conflict issue while still providing the ability to easily see any shifting dependencies upstream.

i.e 

```
# .gitattributes
yarn.lock -diff
* text=auto
bin/* eol=lf
```